lib/pengine/unpack.c

```diff
@@ -72,15 +72,20 @@ pe_fence_node(pe_working_set_t * data_set, node_t * node, const char *reason)
         resource_t *rsc = node->details->remote_rsc->container;
 
         if (is_set(rsc->flags, pe_rsc_failed) == FALSE) {
-            crm_warn("Guest node %s will be fenced (by recovering %s) %s",
-                node->details->uname, rsc->id, reason);
+            if (!is_set(rsc->flags, pe_rsc_managed)) {
+                crm_notice("Not fencing guest node %s because the container is "
+                           "unmanaged", node->details->uname);
```

Comment:
After not seeing the log in my tests I'm actually quite sure - but shouldn't hurt either ...